### PR TITLE
feat: add nfpm meta mackages

### PR DESCRIPTION
**Pull request by reddec**
Allows to use "meta" package in NFPM: packages without binaries

Related to https://github.com/goreleaser/goreleaser/discussions/1612

### CODE REVIEW

**Comment by reddec** [comment]
Sorry, I forgot. Give me a minute!

**Comment by reddec** [comment]
@caarlos0 , done

File: www/docs/customization/nfpm.md [hunk 1]
```diff
@@ -95,6 +95,11 @@ nfpms:
     # Defaults to empty.
     release: 1
 
+    # Makes meta package - empty (without binaries)
+    # package that contains only supporting files and dependencies.
```
Comment (caarlos0) [inline www/docs/customization/nfpm.md:99]:
```suggestion
    # Makes a meta package - an empty package that contains only supporting files and dependencies.
    # When set to `true`, the `builds` option is ignored.
```

**Comment by caarlos0** [comment]
Thanks! 🤘

**Pull request merged**